src/features/customerList/utils.ts

```diff
@@ -75,3 +41,39 @@ function getBerthLeases(profile: ProfileNode): CustomerListBerthLeases[] | undef
       };
     });
 }
+
+export const getCustomersData = (data: CUSTOMERS | undefined): CustomerData[] => {
+  return (
+    data?.profiles?.edges.reduce<CustomerData[]>((acc, profileEdge) => {
+      if (profileEdge?.node) {
+        const profile = profileEdge.node;
+        const boats = getBoats(profile) ?? [];
+        const applications = getApplications(profile) ?? [];
+        const berthLeases = getBerthLeases(profile) ?? [];
+
+        const { organization, primaryAddress } = profile;
+
+        const profileData: CustomerData = {
+          address: organization?.address ?? primaryAddress?.address,
```

Comment:
could organization.address be empty string? Does this work like expected then?

Reply:
The nullish coalescing operator still returns the left operand if it is an empty string, so this works exactly like intended: if the customer profile has an organization, the organization's address will be shown, even if it is empty.

Reply:
Yeah, just making sure. I wanted to know if organization.address is empty string, do we want to use primaryAddress.address.